CHANGELOG

```diff
@@ -1,5 +1,17 @@
+v0.5.0 (3/22/16)
+* Added `scroll_to_element(element_selector)`
+* Make bok-choy more resilient to transient webdriver failures
+* Disable default promo tabs on firefox profile
+* Optional XSS vulnerability check
```

Comment:
nit: that might minimize the new feature's usefulness. Maybe "Implement feature for XSS (cross-site scripting) vulnerability auditing"